data/groups.yaml

```diff
@@ -11,18 +11,51 @@ gimp:
     - pygtk2
     - pycairo
 olpc:
-    name: OLPC/Sugar
+    name: Sugar desktop environment
+    exception: https://pagure.io/fesco/issue/2267
     hidden: true
     packages:
-    # dnf repoquery --repo rawhide 'sugar-*' 'olpc-*' --qf '- %{name}'
     - olpc-kbdshim
-    - olpc-netutils
```

Comment:
I think this should remain here.It is a Python 2 OLPC package, it just has x86_64 excluded, so it sometimes doesn't show up.